mappings/net/minecraft/client/gui/widget/list/EntryListWidget.mapping

```diff
@@ -84,6 +88,8 @@ CLASS net/minecraft/unmapped/C_nmdfcqgq net/minecraft/client/gui/widget/list/Ent
 		ARG 1 graphics
 		ARG 2 x
 		ARG 3 y
+	METHOD m_tpmgwusc isZero (I)Z
+		ARG 1 i
```

Comment:
`index`?

Reply:
The method body is just `return i==0` so I'm not sure what to name the parameter really..